capstone-api/models/User.js

```diff
@@ -114,7 +112,7 @@ class User {
     let followers = await this.getFollowers(username);
     let result = [];
```

Comment:
let's avoid using result-- what does this represent?

Reply:
addressed in later PR